sdk/storage/azure-storage-file-share/tests/test_directory_async.py

```diff
@@ -494,6 +494,52 @@ async def test_list_subdirectories_and_files_async(self, storage_account_name, s
         self.assertEqual(list_dir[5]['is_directory'], False)
         self.assertEqual(list_dir[5]['size'], 5)
 
+    @FileSharePreparer()
+    @AsyncStorageTestCase.await_prepared_test
```

Comment:
nit: You don't really need this line even for non test-proxy files but since this will migrate to test proxy soon, fine to leave it :)